src/notifyGeneralChannel.ts

```diff
@@ -0,0 +1,15 @@
+import { DMChannel, GuildChannel, TextChannel } from "discord.js";
+
+export const notifyGeneralChannel = (channel: DMChannel | GuildChannel) => {
+    const channelName = 'general';
+    const generalChannel = channel.client.channels.cache.find(currentChannel =>
+        currentChannel.id === process.env.GENERAL_CHANNEL_ID
```

Comment:
It might be worth defaulting this to `general` if no value is set?

Reply:
Yes, but in that case, we should try to compare the `currentChannel.name` with 'general', since the id is a number I believe. How about we change to this:
```ts
currentChannel.id === process.env.GENERAL_CHANNEL_ID ||
(currentChannel instanceof TextChannel && currentChannel.name === channelName)
```
If we don't have `currentChannel instanceof TextChannel`TS will give an error since that property isn't on the `Channel` type 😅.

Reply:
Sounds good 👍